### PR TITLE
Rspec Test of Advanced Building Blocks - Enumerables

### PR DESCRIPTION
**Pull request by charlyeneh**
In this project, we tries to apply all the necessary Rspec test for Advanced Building Blocks - Enumerables. 
This was done by applying various scenarios to the individual enumerable methods.

### CODE REVIEW

File: enumerables.rb [hunk 1]
```diff
@@ -117,43 +116,9 @@ def multiply_els(arr)
...
-p example_array.my_inject(:*) # 31616118072
-
-p multiply_els(example_array) # 31616118072
-
-p 'Testing my any edge cases after TSE review'
-p [1, false, nil].my_any? # true
-p [nil, false, nil].my_any? # false
-p [1, false, nil].any? # true
-p [nil, false, nil].any? # false
+# p [1,2,3,4,5,8,8,6].my_count { |value| value %2==0 }
```
Comment (cyonii) [inline enumerables.rb]: Please remove commented code

**Review by cyonii — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️✍️

You have done very nice work so far, I'll like to commend your energy. 👏

- Great having your linter passing
- You did a fantastic job keeping your project clean.

## To Improve
- [ ]  You should not comment pieces of code in your repo. [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
- [ ] `#multiply_els` does not seem to have been tested, please write tests for this function
- [ ]  You have done great adding these tests but you have not tested for much of the possible scenarios. Think of all the possible contexts for each of the enumerable method and add tests for them.

## Nice To Have (Optional)
These are optional suggestions which means you do not have to implement them, although I recommend you do.
- [ ] You have altered your README file clearing old descriptions and instructions about this repo. Remember the whole repo is more about enumerable methods and not RSpec, I suggest you replace your alterations and instead include a section that talks about how to run tests/Rspec.

> ### You can find me on slack 👉 [CY Kalu](https://microverse-students.slack.com/team/U01494U96FR) if you need further assistance or clarification. I'm always available and will be glad to help.

**Review by GzimAsani — APPROVED** [review]
# STATUS: Project Accepted :green_circle: 

### Hello team, great job implemented the required changes from the previous review, your project is accepted and you may merge this pull request

_**Don't forget to submit project completion form**_

**Comment by promise-J** [comment]
GzimAsani thank you so much.

**Pull request merged**